### DISCUSSION TITLE
Which tool is used by security onion to break pcap

### DISCUSSION THREAD

**Discussion by cybersecurity99** [Security-Onion-Solutions/securityonion#7698] [Unsupported Versions] [answered]
I want to know how SO internally breaks pcap into packets .
Like we can see alerts in SO and download related PCAP to that

**Comment by tomjones1977** [comment, marked as answer]
The docs explain the ingest process for network traffic - different tools give you different visibility:

https://docs.securityonion.net/en/2.3/network.html
